Team6 added voting chapter

walkeronwater opened the pull request:
TODO: reference to other chapters' labels

walkeronwater committed:
Team6 added voting chapter

TODO: reference to other chapters' labels

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+\section{Voting Scenarios}
+\label{sec:VotingScenarios}
+
+The voting is called when a collective decision is needed to make in a justified way. A robust voting system is designed for the organisational meetings especially for IIGO in this coursework. % TODO ref to IIGO chapter
```
suggested re-writing (second sentence):

**Within the context of the coursework, robust voting systems are designed for deciding the outcome of collective actions (e.g. elections for legislative, executive and judicial branches in IIGO).**

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+\label{sec:VotingScenarios}
+
+The voting is called when a collective decision is needed to make in a justified way. A robust voting system is designed for the organisational meetings especially for IIGO in this coursework. % TODO ref to IIGO chapter
+When a voting is called in IIGO, the roles President, Speaker, Judge, and Voters will be empowered to have relevant functions to proceed a series of action:
```
suggested re-writing:

When **voting** is called in IIGO, **the President, Speaker, Judge and the islands in the archipelago (i.e. the \emph{Voters}, assuming that no diplomatic sanction is in place) will be empowered to proceed with the following series of actions**:

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+The voting is called when a collective decision is needed to make in a justified way. A robust voting system is designed for the organisational meetings especially for IIGO in this coursework. % TODO ref to IIGO chapter
+When a voting is called in IIGO, the roles President, Speaker, Judge, and Voters will be empowered to have relevant functions to proceed a series of action:
+\begin{itemize}
+    \item President will be eligible to select a motion for the archipelago to vote for.
```
\item **The** President will be eligible to select a **rule** for the archipelago to vote for.

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+When a voting is called in IIGO, the roles President, Speaker, Judge, and Voters will be empowered to have relevant functions to proceed a series of action:
+\begin{itemize}
+    \item President will be eligible to select a motion for the archipelago to vote for.
+    \item Speaker will be eligible to take charge of the proceeding of voting i.e. receiving ballots, counting ballots, and declare the winner.
```
\item **The Speaker will be eligible to take charge of enabling the voting procedure by receiving, counting ballots and ultimately, declaring the winner.**

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+\begin{itemize}
+    \item President will be eligible to select a motion for the archipelago to vote for.
+    \item Speaker will be eligible to take charge of the proceeding of voting i.e. receiving ballots, counting ballots, and declare the winner.
+    \item Judge will be eligible to inspect ballots and the Speaker's action. % ref to IIGO judge inspectBallot
```
No, this is NO MORE the case with accountability cycle introduced in post-mvp. 

**delete this item**

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+    \item President will be eligible to select a motion for the archipelago to vote for.
+    \item Speaker will be eligible to take charge of the proceeding of voting i.e. receiving ballots, counting ballots, and declare the winner.
+    \item Judge will be eligible to inspect ballots and the Speaker's action. % ref to IIGO judge inspectBallot
+    \item Voters will be eligible to vote i.e. submit ballots.
```
\item The islands in the archipelago (i.e. Voters) will be eligible to vote by submitting their ballots\footnote{This is based on the assumption that no diplomatic sanctions are in place for any islands.}.

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+    \item Voters will be eligible to vote i.e. submit ballots.
+\end{itemize}
+
+In an IIGO meeting, the elections of roles and changes of rules need voting for such collective decision-making.
```
**In an IIGO session, the voting action is needed for deciding the next President, Speaker, Judge and also for deciding whether or not to adapt a new rule as well as to disable an active one.**

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+In the initial implementation, it is assumed that all 6 islands have the power to vote at any necessary voting scenario and no sanction or power restriction applied to any island when it comes to the right to vote. However, in the final implementation, some complications or dilemma are applied to, i.e. an/some island(s) could lose their right to vote or not permitted to participate in a voting ev ...
+
+\section{Implementation}
+\label{sec:Implementation}
```
can we please NOT have the implementation part now? cuz maybe, (not decided yet) we gonnna include them in a different section :)

so, for now, I'll be only checking the English-written version
and therefore, suggest you ONLY pushing this non-technical part to the main.

walkeronwater replied:
deleted Implementation section

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+
+In an IIGO meeting, the elections of roles and changes of rules need voting for such collective decision-making.
+
+\textbf{Elections of roles:} As per the IIGO specification, the elections of roles will be conducted according to the cycle of accountability i.e. the Judge can initiate the power transfer of the role of President through voting; the President can initiate the power transfer of the role of \texttt{Speaker} through voting; the Speaker can initiate the power transfer of the role of Judge through vo ...
```
suggested re-writing:

\textbf{**Elections of IIGO roles**:} As per the IIGO specification, the elections of roles will be conducted according to **the accountability cycle (see Section XX)**: **The Judge can initiate the transfer-of-power for the Speaker; the President can initiate the transfer-of-power for the Judge; the Speaker can initiate the transfer-of-power for the President. To prevent multiple-role enactment of the same island, the island that has been selected for a role will be removed from the candidate list for other roles in the current turn\footnote{Note that this assumes that the following rule is in place: The President, Speaker and Judge are all different islands.}. Depending on the sequence of the elections called, the candidates for three roles will consist of maximally six, five, and four islands accordingly. The ballots from the Voters will be in the format of preference lists which rank all candidates in the desired order**.

walkeronwater replied:
I'll change 

> (see Section XX)

 when IIGO chapter is added

ezgimez replied:
yep, perfect

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+
+\textbf{Elections of roles:} As per the IIGO specification, the elections of roles will be conducted according to the cycle of accountability i.e. the Judge can initiate the power transfer of the role of President through voting; the President can initiate the power transfer of the role of \texttt{Speaker} through voting; the Speaker can initiate the power transfer of the role of Judge through vo ...
+
+\textbf{Changes of rules:} the President can select a rule from the proposed list to be voted by the Voter islands. The Speaker will host the voting for changes of rules and be responsible for the ballots counting and results declaration. The ballots from the Voter islands will be: Aye, namely agree with the rule change; Nay, namely disagree with the rule change; or Abstain.
```
suggested re-writing:

\textbf{Changes of rules:} **The President has the power to select a rule from the \emph{rule proposal list} to be voted by the Voters. The Speaker will host the voting for this rule and will be responsible for counting the votes in order to declare the result. The votes from the Voters can only be of the following types: \textbf{Aye} (namely agree with the rule change), \textbf{Nay} (namely disagree with the rule change) or \textbf{Abstain}.**

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+
+Table \ref{table:votingmethod} indicates available choices for each voting scenario.
+
+\begin{table}[H]
```
put the table above line 24 so that we make sure that `\ref{table:votingmethod}` absolutely works

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+\section{Voting Methods}
+\label{sec:VotingMethods}
+
+Table \ref{table:votingmethod} indicates available choices for each voting scenario.
```
Table \ref{table:votingmethod} indicates the available voting method choices for each voting scenario considered in the coursework.

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+\end{center}
+\end{table}
+
+\textbf{Plurality} will select the majority choices as the collective decision. When the decision is picking between two options, e.g. whether agree or disagree a change of rule. Unlike the elections, it is not necessary to compare the options left for determining a relative preference. It is straightforward and effective especially when the number of valid ballots (i.e. abstention excluded) is o ...
```
**re-write** \textbf{Plurality} section.
let's not really include subjective comments like 'effective'', ''straightforward''.

it is not MAJORITY, but rather ''relative majority'' --> meaning the majority of the SUBMITTED votes.

walkeronwater replied:
rewrited

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+
+\textbf{Plurality} will select the majority choices as the collective decision. When the decision is picking between two options, e.g. whether agree or disagree a change of rule. Unlike the elections, it is not necessary to compare the options left for determining a relative preference. It is straightforward and effective especially when the number of valid ballots (i.e. abstention excluded) is o ...
+
+\textbf{Borda Count} method will allow voters to rank preference instead of choosing one option only. The order of choices in the preference list determines the Borda points regarding each option. Adding such Borda points from all voters' ballots produces the Borda score for each candidate. Thus, the candidate with the highest Borda score will be the collective decision.
```
suggested re-writing:

**\textbf{Borda Count} method **will allow Voters to rank their** preference instead of choosing \emph{only} one option. The order of choices in the preference list determines the Borda points regarding each option. Adding such Borda points from all votes produces the Borda score for each candidate.** Thus, the candidate with the highest Borda score will be **the winner.** 



note that I write ''Voters'' as I defined this thing in the beginning of the section!

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+%Normally, voters cast a single vote for their preferred candidate, but in our game specification it is also possible for voters to make their preference list.
+Runoff voting is widely recognized and used around the world primarily for the election of the head of state. It is an effective voting method for the selection of candidates.
+
+\texttt{Voter} islands will rank their preference of candidates and submit the preference lists as their ballots. In the first round, two candidates with most first placed votes are chosen among all candidates. The Runoff voting can conclude the election in the first-round if one candidate obtains a majority. Otherwise, the second round need to be considered and utilized where the ballots will be ...
```
why \texttt{} ??

ezgimez replied:
put line 58 just after \textbf{Runoff} to start with explaning the method first.

ezgimez replied:
it is not Runoff voting ,

it is _Runoff voting method_

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+
+\textbf{Runoff} (also known as the two-round system) is a voting method which is used to pick a single winner from a list of several candidates. 
+%Normally, voters cast a single vote for their preferred candidate, but in our game specification it is also possible for voters to make their preference list.
+Runoff voting is widely recognized and used around the world primarily for the election of the head of state. It is an effective voting method for the selection of candidates.
```
give a very specific example. 

around the world is NOT specific enough.

again, plz do NOT say ''effective''

walkeronwater replied:
rewrited

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+
+\texttt{Voter} islands will rank their preference of candidates and submit the preference lists as their ballots. In the first round, two candidates with most first placed votes are chosen among all candidates. The Runoff voting can conclude the election in the first-round if one candidate obtains a majority. Otherwise, the second round need to be considered and utilized where the ballots will be ...
+
+\textbf{Instant Runoff} is a voting method which is used in single-seat elections which consist of more than two candidates. 
```
re-write

**the election for a single seat**

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+\texttt{Voter} islands will rank their preference of candidates and submit the preference lists as their ballots. In the first round, two candidates with most first placed votes are chosen among all candidates. The Runoff voting can conclude the election in the first-round if one candidate obtains a majority. Otherwise, the second round need to be considered and utilized where the ballots will be ...
+
+\textbf{Instant Runoff} is a voting method which is used in single-seat elections which consist of more than two candidates. 
+Individual voters rank their preference of candidates. The candidate who gets the least number of first-placed is eliminated and this step is repeated until only one candidate remains.
```
The candidate who gets the least number of first-placed is eliminated**,** and this step is repeated until only one candidate remains.

do not forget ","

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+
+% In the event of tied result, alternative rules can be chosen to give a single winner. It is obvious that the effectiveness of this method comes from its flexibility of various rules available in a tie situation.
+
+\textbf{Approval} is a voting method for single winner in which each voter selects a subset of candidates. As of its procedures and usage in the game, Approval will be used for the elections of roles where each voter picks a group of candidates and the most named or approved candidate wins.
```
suggested re-writing:

DELETE(As of its procedures and usage in the game). let's not say ''game'' but coursework. 

Approval will be used for the elections of roles where each voter picks a group of candidates**,** and **the most named candidate wins.**

do not forget ","

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+
+\textbf{Approval} is a voting method for single winner in which each voter selects a subset of candidates. As of its procedures and usage in the game, Approval will be used for the elections of roles where each voter picks a group of candidates and the most named or approved candidate wins.
+
+\textbf{Tied situation:} In a tied situation, there are three possible rules that can be used to eliminate one or more candidates who get the same number of votes. The Condorcet method is one of the voting methods which selects the candidate that dominates other candidates in term of majority relation which ranks the candidates according to one-to-one comparisons. In other words, the Condorcet wi ...
```
do note say \textbf{Tied situation} -- this NOT  another method lol.

suggested re-writing:
**In case of a tied election,** there are three possible methods that can be used as a tiebreaker.

please `\begin{itemize} \end{itemize}` these three methods

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+
+\textbf{Tied situation:} In a tied situation, there are three possible rules that can be used to eliminate one or more candidates who get the same number of votes. The Condorcet method is one of the voting methods which selects the candidate that dominates other candidates in term of majority relation which ranks the candidates according to one-to-one comparisons. In other words, the Condorcet wi ...
+
+In this coursework, the pairwise Condorcet winner is chosen for a tiebreak setup which states that a candidate who dominates the other candidate in term of majority relation wins.
```
suggested re-writing:

In this coursework, the pairwise Condorcet winner is chosen for as **a tiebreaker** which states that a candidate who dominates the other candidate in term of majority relation wins.

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+\node[draw, rounded corners] (start)  {The President selects a rule to be voted on};
+\node[draw, below=of start] (step 1)  {The President notifies the Speaker to hold a voting session on the selected rule};
+\node[draw, below=of step 1] (step 2)  {The Speaker opens ballots for all islands to cast the votes};
+\node[draw, below=of step 2] (step 3)  {The Speaker closes ballots after receiving votes from all islands};
```
The Speaker **closes the ballot after receiving votes from the Voters.**

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+\node[draw, below=of start] (step 1)  {The President notifies the Speaker to hold a voting session on the selected rule};
+\node[draw, below=of step 1] (step 2)  {The Speaker opens ballots for all islands to cast the votes};
+\node[draw, below=of step 2] (step 3)  {The Speaker closes ballots after receiving votes from all islands};
+\node[draw, below=of step 3] (step 4)  {The Speaker counts the votes by calling a voting method/function for winner selection};
```
The Speaker counts the votes by **using a specific voting method** to **declare the result.**

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+\node[draw, below=of step 1] (step 2)  {The Speaker opens ballots for all islands to cast the votes};
+\node[draw, below=of step 2] (step 3)  {The Speaker closes ballots after receiving votes from all islands};
+\node[draw, below=of step 3] (step 4)  {The Speaker counts the votes by calling a voting method/function for winner selection};
+\node[draw, below=of step 4, rounded corners] (end)   {The Speaker publicly declares the result of voting};
```
{The Speaker publicly declares the result **for the finalised vote.**

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+\centering
+\node[draw, rounded corners] (start)  {The President selects a rule to be voted on};
+\node[draw, below=of start] (step 1)  {The President notifies the Speaker to hold a voting session on the selected rule};
+\node[draw, below=of step 1] (step 2)  {The Speaker opens ballots for all islands to cast the votes};
```
**opens the ballot for the Voters to cast their vote.**

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+
+\section{Voting Protocol}
+\label{sec:VotingProtocol}
+Voting Protocol is adopted from an example of formal specification given in Robert's Rules of Order (RONR) that defines a series of sequential actions when a voting event happens. This protocol is applicable to any voting method at certain state of the game (i.e. during IIGO session) whenever a voting event happens. With the basis from RONR\footnote{refer to Prof. J Pitt's Draft Book: Self-Organi ...
```
suggested re-writing:

**The suggested voting protocol** is adapted from an example of formal specification .....

This protocol is applicable to any voting method at **a certain** state of the game.

**The sequence of actions of the voting protocol in this game when a voting event happens is as follows: The President selects a rule from the rule proposal list to be voted on**; the President notifies the Speaker to hold a voting event for the selected rule; the Speaker calls for a vote for the selected rule by opening ballots to all islands to cast their votes; the Speaker closes the ballots after receiving votes from all islands; the Speaker counts the votes by calling an applicable voting method function for winner selection to produce a result; **finally**, the Speaker publicly declares the voting result whether the selected rule can be carried out or not. Figure~\ref{fig:RONRVotingProtocol} shows **the sequence** of the voting protocol for changing and/or adapting a new rule.




can you ask Pitt to how to _formally_ cite his 'book'? @walkeronwater

walkeronwater replied:
will do

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,192 @@
+
+For elections of roles, the sequence of actions of the voting protocol is mostly similar to the above explanation in principle, except for some parameters, such as the motion of the vote which is the role itself (President, or Speaker, or Judge), the facilitator of the election/vote event which depends on what role is being held for election (refer to Chapter 5 IIGO for more details on change of  ...
+
+In the initial implementation, it is assumed that all 6 islands have the power to vote at any necessary voting scenario and no sanction or power restriction applied to any island when it comes to the right to vote. However, in the final implementation, some complications or dilemma are applied to, i.e. an/some island(s) could lose their right to vote or not permitted to participate in a voting ev ...
```
comma issue!

suggested-rewriting

In the initial turn of the game, it is assumed that all 6 islands have the power to vote at any necessary voting scenario**,** and **no diplomatic sanction is in place for any island.** **However, in the further turns of the game, some island(s) could lose their right to vote and/or not permitted to participate in a voting event due to the diplomatic sanction(s) in place**. Refer to Chapter 5 IIGO for details on diplomatic sanction. **In this case, the Speaker will open the ballots to the eligible islands only** , **i.e. those who are permitted to vote at a certain state of the game.**

walkeronwater replied:
changed

walkeronwater committed:
Updated chapter voting

changed sentences
removed section "Implementation"
thanks to Ezgi for reviewing!

yp717 commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+\section{Voting Scenarios}
+\label{sec:VotingScenarios}
+
+Within the context of the coursework, robust voting systems are designed for deciding the outcome of collective actions (e.g. elections for legislative, executive and judicial branches in IIGO).
```
technically for academic writing we should be using the passive voice. Suggested re-write: 

Within the context of the coursework, robust voting systems were designed to determine the outcome of collective actions (e.g. elections for legislative, executive and judicial branches in IIGO).

ezgimez replied:
so your suggestion is "are" --> "were" ??
"are designed" is also passive voice. I don't understand your point @yp717

ezgimez replied:
designed for --> designed to .
ok I agree with this.

ezgimez replied:
the sentence 5 is already _passive voice_ . so your point is not right, I think @yp717

walkeronwater replied:
changed

yp717 commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+\label{sec:VotingScenarios}
+
+Within the context of the coursework, robust voting systems are designed for deciding the outcome of collective actions (e.g. elections for legislative, executive and judicial branches in IIGO).
+When voting is called in IIGO, the President, Speaker, Judge and the islands in the archipelago (i.e. the \emph{Voters}, assuming that no diplomatic sanction is in place) will be empowered to proceed with the following series of actions:
```
Given IIGO stands for Inter Island Governmental Organisation should we say the IIXO or IIGO (not major but would be nice to be consistent across the doc) - other doc editors chime in here pls

ezgimez replied:
what? what is IIXO??

ezgimez replied:
btw it is 
"**Inter-Island** Governmental Organisation"

ezgimez replied:
it is ''the IIGO''

walkeronwater replied:
changed - "the IIGO"

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+
+\textbf{Borda Count} method will allow Voters to rank their preference instead of choosing \emph{only} one option. The order of choices in the preference list determines the Borda points regarding each option. Adding such Borda points from all votes produces the Borda score for each candidate. Thus, the candidate with the highest Borda score will be the winner.
+
+\textbf{Runoff} (also known as the two-round system) is a voting method which counts the ballots for two rounds. Voter islands will rank their preference of candidates and submit the preference lists as their ballots. In the first round, candidates with the top-most and second-most first-placed votes are chosen among all candidates. If one of them obtains a majority, the Runoff voting can conclud ...
```
suggested re-writing:

If one of them obtains a majority, **the Runoff method** can conclude **this candidate to be the winner**. Otherwise, the second round will proceed where the ballots will be recounted by eliminating the candidates other than **the top two choices**. Subsequently, the candidate with the most votes between the **top two** candidates will be the winner.

walkeronwater replied:
changed

yp717 commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+    \item The islands in the archipelago (i.e. Voters) will be eligible to vote by submitting their ballots\footnote{This is based on the assumption that no diplomatic sanctions are in place for any islands.}.
+\end{itemize}
+
+In an IIGO session, the voting action is needed for deciding the next President, Speaker, Judge and also for deciding whether or not to adapt a new rule as well as to disable an active one.
```
Different phrasal verb may be more appropriate. Suggested rewrite:

In an IIGO session, the voting action is needed to decide the next President, Speaker, Judge and also to decide whether or not to adapt a new rule as well as to disable an active one.

ezgimez replied:
designed for --> designed to .
ok I agree with this.
(same as above)

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+
+In case of a \textbf{tied election}, there are three possible rules that can be used to eliminate one or more candidates who get the same number of votes:
+\begin{itemize}
+    \item The Condorcet method is one of the voting methods which selects the candidate that dominates other candidates in term of majority relation which ranks the candidates according to one-to-one comparisons. In other words, the Condorcet winner is the candidate that wins a majority in every one-to-one comparison against other candidates. However, Condorcet method is not guaranteed to work wh ...
```
suggested re-writing:

 **in terms** of majority relation which ranks the candidates according to one-to-one comparisons.

... the Condorcet winner is the candidate that wins **the majority** in every one-to-one comparison against other candidates. 

is it really "voting paradox" ? I'm asking cuz maybe it has a more complete name :)

walkeronwater replied:
changed and indeed it is "voting paradox" (at least from Wikipedia - I know that may not sound reliable), had deleted it to prevent confusion though

yp717 commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+
+In an IIGO session, the voting action is needed for deciding the next President, Speaker, Judge and also for deciding whether or not to adapt a new rule as well as to disable an active one.
+
+\textbf{Elections of IIGO roles:} As per the IIGO specification, the elections of roles will be conducted according to the accountability cycle (see Section XX): The Judge can initiate the transfer-of-power for the Speaker; the President can initiate the transfer-of-power for the Judge; the Speaker can initiate the transfer-of-power for the President. To prevent multiple-role enactment of the sam ...
```
Could you change Section XX to correctly reference the appropriate chapter/section

ezgimez replied:
I wrote Section XX.. the section  that @walkeronwater is  referring to  is not present  now -- will come later

walkeronwater replied:
Will do this later when section accountability cycle is available

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+\begin{itemize}
+    \item The Condorcet method is one of the voting methods which selects the candidate that dominates other candidates in term of majority relation which ranks the candidates according to one-to-one comparisons. In other words, the Condorcet winner is the candidate that wins a majority in every one-to-one comparison against other candidates. However, Condorcet method is not guaranteed to work wh ...
+    \item Hare Rule can be used in a tiebreak where all candidates with the least first-place votes will be all eliminated.
+    \item Coombs Rule could be used in place of Hare Rule if the iterative process of removing candidates with the least first-place votes is not preferred. Coombs Rule indicates that the candidate with the most last-placed votes will be removed.
```
suggested re-writing:

Coombs Rule **states** that the candidate with the most last-placed votes will be removed.

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+    \item Coombs Rule could be used in place of Hare Rule if the iterative process of removing candidates with the least first-place votes is not preferred. Coombs Rule indicates that the candidate with the most last-placed votes will be removed.
+\end{itemize}
+
+In this coursework, the pairwise Condorcet winner is chosen for as a tiebreaker which states that a candidate who dominates the other candidate in term of majority relation wins.
```
In this coursework, the pairwise Condorcet winner **is chosen as a tiebreaker** which states that a candidate who dominates the other candidate **in terms** of majority relation wins.

walkeronwater replied:
changed

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+
+\section{Voting Protocol}
+\label{sec:VotingProtocol}
+The suggested voting protocol is adapted from an example of formal specification given in Robert's Rules of Order (RONR) that defines a series of sequential actions when a voting event happens.
```
where is this ''example''??

walkeronwater replied:
From Prof Pitt's book/paper, will add a reference to it

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+\end{center}
+\end{figure}
+
+For elections of roles, the sequence of actions of the voting protocol is mostly similar to the above explanation in principle, except for some parameters, such as the motion of the vote which is the role itself (President, or Speaker, or Judge), the facilitator of the election/vote event which depends on what role is being held for election (refer to Chapter 5 IIGO for more details on change of  ...
```
loved this paragraph --- good wording

ezgimez commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+
+For elections of roles, the sequence of actions of the voting protocol is mostly similar to the above explanation in principle, except for some parameters, such as the motion of the vote which is the role itself (President, or Speaker, or Judge), the facilitator of the election/vote event which depends on what role is being held for election (refer to Chapter 5 IIGO for more details on change of  ...
+
+In the initial implementation, it is assumed that all 6 islands have the power to vote at any necessary voting scenario, and no diplomatic sanction is in place for any island. However, in the further turns of the game, some island(s) could lose their right to vote and/or not permitted to participate in a voting event due to the diplomatic sanction(s) in place. Refer to Chapter 5 IIGO for details  ...
```
suggested re-writing: 

**At the start of the game,**  it is assumed that...

walkeronwater replied:
changed

yp717 commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+
+Table \ref{table:votingmethod} indicates the available voting method choices for each voting scenario considered in the coursework.
+
+\textbf{Plurality} will select the relative majority choices as the collective decision. In the IIGO session, the Plurality voting method will be used to decide whether a change of rule goes into effect. When the number of valid ballots (i.e. abstention excluded) is odd, the Plurality method can determine the winner without tied results. Otherwise, the solution to tied results is regimented with  ...
```
dangling modifier + reference to IIGO as if it only happens once using "the". Suggested rewrite for second sentence:

The Plurality voting method will be used to decide whether a change of rule goes into effect in an IIGO session.

yp717 replied:
i.e. seems to be incorrectly punctuated. Suggested rewrite: 
(i.e., abstention excluded)

ezgimez replied:
better wording, agreed

ezgimez replied:
dangling modifier --> what you mean?

yp717 replied:
its the technical name of the grammatical structure when you have a sub-clause hanging on the front of the sentence, and a clause after it

walkeronwater replied:
changed

yp717 commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+
+\textbf{Borda Count} method will allow Voters to rank their preference instead of choosing \emph{only} one option. The order of choices in the preference list determines the Borda points regarding each option. Adding such Borda points from all votes produces the Borda score for each candidate. Thus, the candidate with the highest Borda score will be the winner.
+
+\textbf{Runoff} (also known as the two-round system) is a voting method which counts the ballots for two rounds. Voter islands will rank their preference of candidates and submit the preference lists as their ballots. In the first round, candidates with the top-most and second-most first-placed votes are chosen among all candidates. If one of them obtains a majority, the Runoff voting can conclud ...
```
Pronoun error. Recommend replacing which with that. Suggested re-write:

\textbf{Runoff} (also known as the two-round system) is a voting method that counts the ballots for two rounds. Voter islands will rank their preference of candidates and submit the preference lists as their ballots.

yp717 replied:
Suggested rewrite. Think top-two should omit the - here.

walkeronwater replied:
changed

yp717 commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+
+\textbf{Runoff} (also known as the two-round system) is a voting method which counts the ballots for two rounds. Voter islands will rank their preference of candidates and submit the preference lists as their ballots. In the first round, candidates with the top-most and second-most first-placed votes are chosen among all candidates. If one of them obtains a majority, the Runoff voting can conclud ...
+
+\textbf{Instant Runoff} is a voting method which is used in the elections for a single seat which consist of more than two candidates. 
```
Possible wordy sentence. Suggested rewrite:

\textbf{Instant Runoff} is a voting method used in elections for a single seat, that consist of more than two candidates.

walkeronwater replied:
changed

yp717 commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+
+\textbf{Approval} is a voting method for a single winner in which each voter selects a subset of candidates. Approval will be used for the elections of roles where each voter picks a group of candidates, and the most named or approved candidate wins.
+
+In case of a \textbf{tied election}, there are three possible rules that can be used to eliminate one or more candidates who get the same number of votes:
```
Suggested rewrite: Replace "there are three possible rules that can be used to" with

three possible rules can

walkeronwater replied:
changed

yp717 commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+
+In case of a \textbf{tied election}, there are three possible rules that can be used to eliminate one or more candidates who get the same number of votes:
+\begin{itemize}
+    \item The Condorcet method is one of the voting methods which selects the candidate that dominates other candidates in term of majority relation which ranks the candidates according to one-to-one comparisons. In other words, the Condorcet winner is the candidate that wins a majority in every one-to-one comparison against other candidates. However, Condorcet method is not guaranteed to work wh ...
```
Suggested Rewrite. However the Condorcet instead of just Condorcet because you are referring to the method

ezgimez replied:
there should be a comma after "however"

walkeronwater replied:
changed

yp717 commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+\begin{itemize}
+    \item The Condorcet method is one of the voting methods which selects the candidate that dominates other candidates in term of majority relation which ranks the candidates according to one-to-one comparisons. In other words, the Condorcet winner is the candidate that wins a majority in every one-to-one comparison against other candidates. However, Condorcet method is not guaranteed to work wh ...
+    \item Hare Rule can be used in a tiebreak where all candidates with the least first-place votes will be all eliminated.
+    \item Coombs Rule could be used in place of Hare Rule if the iterative process of removing candidates with the least first-place votes is not preferred. Coombs Rule indicates that the candidate with the most last-placed votes will be removed.
```
Suggested rewrite. last-placed doesn't seem to fit the context I think this should say last-place

yp717 replied:
also not sure if the dash is necessary do we have a convention here since there are a few of these @ezgimez

ezgimez replied:
you are welcome to come up with your convention if you deem it is necessary @yp717 .
the dash should be more of common sense -- I don't think there is any _convention_  we can follow

yp717 replied:
agreed I don't think its a major thing but the last-placed to last-place is quite important

walkeronwater replied:
changed - I follow the convention of using "last-place"

yp717 commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+    \item Coombs Rule could be used in place of Hare Rule if the iterative process of removing candidates with the least first-place votes is not preferred. Coombs Rule indicates that the candidate with the most last-placed votes will be removed.
+\end{itemize}
+
+In this coursework, the pairwise Condorcet winner is chosen for as a tiebreaker which states that a candidate who dominates the other candidate in term of majority relation wins.
```
Suggested rewrite. 

In this coursework, the pairwise Condorcet winner is chosen as a tiebreaker that states that a candidate who dominates the other candidate in terms of majority relation wins.

ezgimez replied:
i did the very same edit lol :D

walkeronwater replied:
changed - sorry for the typos

yp717 commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+
+This protocol is applicable to any voting method at a certain state of the game.
+
+The sequence of actions of the voting protocol in this game when a voting event happens is as follows: The President selects a rule from the rule proposal list to be voted on; the President notifies the Speaker to hold a voting event for the selected rule; the Speaker calls for a vote for the selected rule by opening ballots to all islands to cast their votes; the Speaker closes the ballots after ...
```
This is an incredibly long sentence with lots of clauses linked by semicolons. It would be far clearer imo if you used a list here to separate this out for the reader and show that it is a series of steps/sequence of actions

ezgimez replied:
hm.. maybe we can do `\begin{itemize} .. \end{itemize}` or smt like that

walkeronwater replied:
changed

yp717 commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+The sequence of actions of the voting protocol in this game when a voting event happens is as follows: The President selects a rule from the rule proposal list to be voted on; the President notifies the Speaker to hold a voting event for the selected rule; the Speaker calls for a vote for the selected rule by opening ballots to all islands to cast their votes; the Speaker closes the ballots after ...
+
+
+\begin{figure}[H]
```
depending on where in the doc this is [H] might shift back and forth - if you want to force it in place would recommend using [!htb] to put it exactly here in your sequence of the doc

walkeronwater replied:
changed

yp717 commented on 08_voting/index.tex:
```diff
@@ -1 +1,101 @@
+\end{center}
+\end{figure}
+
+For elections of roles, the sequence of actions of the voting protocol is mostly similar to the above explanation in principle, except for some parameters, such as the motion of the vote which is the role itself (President, or Speaker, or Judge), the facilitator of the election/vote event which depends on what role is being held for election (refer to Chapter 5 IIGO for more details on change of  ...
```
this should be a reference to the chapter using the latex command - if the chapters are reordered and we miss something like this it could get really confusing and missed before submission

ezgimez replied:
so... what is wrong with this `ref{}` I don't understand your comment @yp717

yp717 replied:
there wasn't a ref before he already changed it I believe

yp717 replied:
nothing is wrong with that ref its perfect

yp717 reviewed (changes requested):
Left a lot of comments I hope it helps :)

ezgimez commented:
sorry @walkeronwater  -- but myself and @yp717  are learning how to review properly :D  we just wanna make sure that the style is consistent across the _whole document_

bon appetit with the edits...

walkeronwater committed:
Update index.tex

ezgimez reviewed (approved):
thanks dear @walkeronwater  for your patience.

plz let's wait @yp717 's approval  BEFORE merging
